### PR TITLE
RTC-S1 Gamma Integration

### LINKED ISSUE #259: L2_RTC_S1 Gamma SAS Integration
Ticket to track integration of the latest "gamma" release of the RTC-S1 SAS with the PGE library.

- [x] Update AF path in RTC-S1 container build script to point at gamma SAS image
- [x] Integrate updated SAS yamale schema
- [x] Update example RTC-S1 PGE/SAS runconfig
- [x] Update file naming conventions to include static layers and browse image
- [x] Update ISO xml template with new/update metadata fields
- [x] Update unit tests where applicable
- [x] Create new expected data sets for use with int test pipeline
- [x] Update ICS document
- [x] Create release branch/release page

### PR DESCRIPTION
**Pull request by collinss-jpl**
## Description
This branch updates the RTC-S1 PGE code to integrate with the Gamma delivery of the corresponding SAS. Major changes include adding file naming support for the new "static layers" and "browse" products, and minor updates to the ISO xml metadata.

## Affected Issues
Resolves #259 

## Testing
Existing unit test suite for CSLC-S1 has been updated to test file name application to the new output products.

### CODE REVIEW

File: src/opera/pge/rtc_s1/rtc_s1_pge.py [hunk 1]
```diff
@@ -249,6 +260,49 @@ def _rtc_filename(self, inter_filename):
...
+        Parameters
+        ----------
+        inter_filename : str
+            The intermediate filename of the output product to generate
+            a filename for. This parameter may be used to inspect the file
+            in order to derive any necessary components of the returned filename.
+
+        Returns
+        -------
+        static_layer_filename : str
```
Comment (david-inglish) [inline src/opera/pge/rtc_s1/rtc_s1_pge.py:285]: this doesn't quite match the return

File: src/opera/pge/rtc_s1/rtc_s1_pge.py [hunk 2]
```diff
@@ -285,6 +339,34 @@ def _rtc_geotiff_filename(self, inter_filename):
...
+        Parameters
+        ----------
+        inter_filename : str
+            The intermediate filename of the output product to generate
+            a filename for. This parameter may be used to inspect the file
+            in order to derive any necessary components of the returned filename.
+
+        Returns
+        -------
+        browse_filename : str
```
Comment (david-inglish) [inline src/opera/pge/rtc_s1/rtc_s1_pge.py:362]: I don't know what the convention is where an anonymous variable is invovled, maybe just put the type?

**Review by david-inglish — APPROVED** [review]
Looks good.

File: src/opera/pge/rtc_s1/rtc_s1_pge.py [hunk 3]
```diff
@@ -249,6 +260,51 @@ def _rtc_filename(self, inter_filename):
...
+
+        Where <RTC filename> is returned by RtcS1PostProcessorMixin._rtc_filename(),
+        and <Static layer name> is the identifier for the specific static layer
+        as parsed from the intermediate filename.
+
+        Parameters
+        ----------
+        inter_filename : str
+            The intermediate filename of the output product to generate
+            a filename for. This parameter may be used to inspect the file
```
Comment (JimHofman) [inline src/opera/pge/rtc_s1/rtc_s1_pge.py:280]: Again.  I think the first sentence needs a little work.

File: src/opera/pge/rtc_s1/rtc_s1_pge.py [hunk 4]
```diff
@@ -249,6 +260,51 @@ def _rtc_filename(self, inter_filename):
 
         return rtc_filename
 
+    def _static_layer_filename(self, inter_filename):
+        """
+        Returns the file name to use for the static layer RTC products produced
```
Comment (JimHofman) [inline src/opera/pge/rtc_s1/rtc_s1_pge.py:265]: I'm being a nit-picker, but maybe:  Returns the file name of the static layer.....

File: src/opera/pge/rtc_s1/rtc_s1_pge.py [hunk 5]
```diff
@@ -285,6 +341,36 @@ def _rtc_geotiff_filename(self, inter_filename):
 
         return f"{rtc_filename}_{polarization}.tif"
 
+    def _browse_filename(self, inter_filename):
+        """
+        Returns the file name to use for the PNG browse image product produced
+        by this PGE.
```
Comment (JimHofman) [inline src/opera/pge/rtc_s1/rtc_s1_pge.py:347]: Again, "Returns the file name of the ...."

**Pull request merged**